### PR TITLE
Added CertMagic for TLS support

### PR DESCRIPTION
**Pull request by fillup**
Also refactored AWS credentials in environment to use same key/secret for S3/SES and now DynamoDB for CertMagic. 

Related terraform PR: https://github.com/silinternational/wecarry-terraform/pull/24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,3 +167,25 @@ will erase or set to `null`. Note that this does not address the
 GraphQL spec requirement to not modify an omitted field. The reference UI 
 implementation, [wecarry-ui](https://github.com/silinternational/wecarry-ui),
 will always include all supported fields.
+
+## TLS / HTTPS
+
+WeCarry API can either run with HTTPS/TLS or standard HTTP. If running behind a load balancer that terminates SSL/TLS 
+connections for you, using HTTPS/TLS in the container as well may not be necessary, but if possible it's a good 
+idea. WeCarry API uses [CertMagic](https://github.com/caddyserver/certmagic) to work it's magic with Let's Encrypt
```
Comment (Baggerone) [inline README.md]: "... to work its magic ..."  (no apostrophe)

File: application/main.go [hunk 2]
```diff
@@ -25,9 +34,50 @@ func main() {
 	rollbar.SetCodeVersion(GitCommitHash)
 	rollbar.SetServerRoot(domain.Env.RollbarServerRoot)
 
+	var srv servers.Server
+	if domain.Env.DisableTLS {
+		srv = servers.New()
+	} else {
+		certmagic.Default.Storage = &dynamodbstore.Storage{
```
Comment (Baggerone) [inline application/main.go]: Maybe put the contents of the `else` block into a separate function or twp?

File: README.md [hunk 3]
```diff
@@ -167,3 +167,25 @@ will erase or set to `null`. Note that this does not address the
 GraphQL spec requirement to not modify an omitted field. The reference UI 
 implementation, [wecarry-ui](https://github.com/silinternational/wecarry-ui),
 will always include all supported fields.
+
+## TLS / HTTPS
+
+WeCarry API can either run with HTTPS/TLS or standard HTTP. If running behind a load balancer that terminates SSL/TLS 
+connections for you, using HTTPS/TLS in the container as well may not be necessary, but if possible its a good
```
Comment (Schparky) [inline README.md]:
```suggestion
connections for you, using HTTPS/TLS in the container as well may not be necessary, but if possible it's a good 
```

File: .env.example [hunk 4]
```diff
@@ -67,3 +71,8 @@ SERVICE_INTEGRATION_TOKEN=
 
 # Maximum number of files to delete in service file_cleanup task
 #MAX_FILE_DELETE=10
+
+# For CertMagic / Let's Encrypt
+CERT_DOMAIN_NAME=
+CLOUDFLARE_AUTH_EMAIL=
+CLOUDFLARE_AUTH_KEY=
```
Comment (Schparky) [inline .env.example]:
```suggestion
CLOUDFLARE_AUTH_KEY=
DYNAMO_DB_TABLE=CertMagic
```

**Pull request merged**